sourcegraph/sourcegraph
# [Cody] add callout for scope selector

closes: #54007
closes: #54012

- Adds dismissable callout for scope selector
- Add settings page link to embeddings status icon
- Fix bug where chat id in the URL for /cody was getting updated even when the user was logged out or cody was disabled.

<img width="1808" alt="image" src="https://github.com/sourcegraph/sourcegraph/assets/22571395/6e86e68a-1c4e-493b-9807-6ebdea225a80">

## Test plan

- visit /cody
- dismiss callout or open scope selector should be dismissed
- reload, should not appear again

- open scope selector
- click on the embeddings status icon, which should open the embeddings settings page in a new tab
- the repo should not be selected/deselected.
